molecule/config.py

```diff
@@ -102,6 +103,22 @@ def after_init(self):
     def write(self):
         util.write_file(self.config_file, util.safe_dump(self.config))
 
+    @property
+    def ansible_version(self):
+        """Return current version of ansible."""
+        return pkg_resources.parse_version(
+            pkg_resources.get_distribution("ansible").version
+        )
+
+    @property
+    def ansible_collections_path(self):
+        """Return collection path variable for current version of Ansible."""
+        # https://github.com/ansible/ansible/pull/70007
+        if self.ansible_version >= pkg_resources.parse_version("2.10"):
```

Comment:
```suggestion
        if self.ansible_version >= pkg_resources.parse_version("2.10.0.dev"):
```
Otherwise `stable-2.10` currently doesn't pass this test (with version `2.10.0.dev0`).